packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts

```diff
@@ -171,14 +170,16 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Ethereum> {
       const data = await getErc20Data(to, tx?.value, erc20ContractAddress)
 
       const txToSign: ETHSignTx = {
-        addressNList,
         value: numberToHex(isErc20Send ? '0' : tx?.value),
         to: destAddress,
         chainId: 1, // TODO: implement for multiple chains
         data,
         nonce: numberToHex(chainSpecific.nonce),
-        gasPrice: numberToHex(gasPrice),
-        gasLimit: numberToHex(gasLimit)
+        gasLimit: numberToHex(gasLimit),
+        maxFeePerGas: numberToHex(maxFeePerGas),
+        maxPriorityFeePerGas: numberToHex(maxPriorityFeePerGas),
+        accessList: [],
+        type: "0x02"
```

Comment:
```suggestion
        gasLimit: gasLimit !== undefined ? numberToHex(gasLimit) : undefined,
        maxFeePerGas: maxFeePerGas !== undefined ? numberToHex(maxFeePerGas) : undefined,
        maxPriorityFeePerGas: maxPriorityFeePerGas !== undefined ? numberToHex(maxPriorityFeePerGas) : undefined
```

we do have to be sure to plumb through `gasPrice` to support legacy TXs if that's what's requested.